lib/rouge/lexers/opentype_feature_file.rb

```diff
@@ -77,34 +85,20 @@ def self.keywords
         rule %r/\s+/, Text::Whitespace
         rule %r/\)/, Punctuation, :pop!
         rule %r/\(/, Punctuation
-        rule %r/[a-z0-9\/_.]*/i, Str
+        rule %r/[a-z0-9\/_.-]*/i, Str
       end
 
       state :strings do
         rule %r/(\([a-z0-9_]+\))?[-#0 +]*([0-9]+|[*])?(\.([0-9]+|[*]))?/i, Str
       end
 
       state :strings_double do
-        rule %r/[^\\"%\n]+/, Str
+        rule %r/[^"%\n]+/, Str
```

Comment:
Doing this means we no longer have any escaped characters within a double-quoted string. Are there really no escaped characters?

Reply:
Not that I can think of. In double-quoted strings there can be escaped sequences, but not single characters. For example, to escape `"` you need to put `\0022` which we could treat separately, but personally I didn't like having those visually separate. I can see what other people on the team think.

Reply:
Ok, I checked with a few people and we can't see any real reason to differentiate escapes in a string.

Reply:
I was going to say that I thought from an accuracy perspective that it is more correct to use the `Str::Escape` token but after looking myself again at the visual sample, I think I agree that in case, it makes sense to treat it the same as a regular character.

I have consolidated the string states together but not in a way that makes any changes to how the rules work.